### PR TITLE
Prepare design system for Print Statements v3

### PR DESCRIPTION
**Pull request by jonbell-lot23**
As part of [DES-220](https://linear.app/replay/issue/DES-220/final-redlines-for-print-statements-30), I set up some new variables so Brian has everything he needs when he does Print Statements v3 work.

[Here's a link to the Figma](https://www.figma.com/file/mAotNZRv6M5X0zHIFKPzFY/Replay-Design-Doc?node-id=15911%3A96426&t=u1kWVzfXEfgEcZuu-4) where I spec'd all this out:

image

### CODE REVIEW

File: packages/replay-next/components/sources/PointPanel.module.css [hunk 1]
```diff
@@ -69,6 +69,10 @@
   gap: 1ch;
 }
 
+.ContentWrapper:hover {
+  border: 2px solid var(--point-panel-input-border-color-hover);
```
Comment (bvaughn) [inline packages/replay-next/components/sources/PointPanel.module.css:73]: I like following this approach for styles like this (so we don't see any unexpected layout shift or clipping as a result of the hover):
```css
.ClassName {
  border: 2px solid transparent;
}
.ClassName:hover {
  border-color: var(--whatever);
}
```

It looks like this component _does_ follow that pattern– but redeclaring the whole `border` shorthand still _could_ make it easier for the two to fall out of sync, so small suggested change:
```suggestion
  border-color: var(--point-panel-input-border-color-hover);
```

**Review by bvaughn — APPROVED** [review]
Cool! thanks!

File: packages/replay-next/components/Icon.tsx [hunk 2]
```diff
@@ -90,6 +91,19 @@ export default function Icon({
       path =
         "M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z";
       break;
+    case "check":
+      path = (
+        <>
+          <path
+            d="M13 8.95508L7.90909 13.9551L6 12.0801"
+            stroke="white"
```
[the remark this answers is not in the document]
Comment (bvaughn) [reply packages/replay-next/components/Icon.tsx:99]: How would you feel about this instead?

```jsx
<path
  d="M21,7L9,19L3.5,13.5L4.91,12.09L9,16.17L19.59,5.59L21,7Z"
  fill="none"
  stroke="currentColor"
  strokeWidth="3"
  strokeLinecap="round"
  strokeLinejoin="round"
/>
```

Shown as the 3rd option below:
Screen Shot 2023-01-17 at 10 17 48 AM

[the remark this answers is not in the document]
Comment (jonbell-lot23) [reply packages/replay-next/components/Icon.tsx:99]: Oh, bless you. That's exactly what I was hoping for and I appreciate the notes for how you got there. Yes, that looks perfect!

**Comment by jonbell-lot23** [comment]
> Ooops I shouldn't have clicked _approve_ until we sorted out the icon/path issue. Let's chat!

I threw in the fixed SVG path, so we should be good to go now.

File: packages/replay-next/components/sources/PointPanel.module.css (hunk 1 above)
Comment (jonbell-lot23) [reply packages/replay-next/components/sources/PointPanel.module.css]: Good call, thanks!

**Review by bvaughn — APPROVED** [review]
🥳

**Pull request merged**